crates/integration_tests/tests/shared_tests/scan_all_type.rs

```diff
@@ -40,29 +40,21 @@ use iceberg::writer::file_writer::location_generator::{
 };
 use iceberg::writer::file_writer::ParquetWriterBuilder;
 use iceberg::writer::{IcebergWriter, IcebergWriterBuilder};
-use iceberg::{Catalog, Namespace, NamespaceIdent, TableCreation};
-use iceberg_integration_tests::set_test_fixture;
+use iceberg::{Catalog, TableCreation};
+use iceberg_catalog_rest::RestCatalog;
 use parquet::arrow::PARQUET_FIELD_ID_META_KEY;
 use parquet::file::properties::WriterProperties;
 use uuid::Uuid;
 
+use crate::get_shared_containers;
+use crate::shared_tests::apple_ios_ns;
+
 #[tokio::test]
 async fn test_scan_all_type() {
-    let fixture = set_test_fixture("test_scan_all_type").await;
-
-    let ns = Namespace::with_properties(
-        NamespaceIdent::from_strs(["apple", "ios"]).unwrap(),
-        HashMap::from([
-            ("owner".to_string(), "ray".to_string()),
-            ("community".to_string(), "apache".to_string()),
-        ]),
-    );
+    let fixture = get_shared_containers();
+    let rest_catalog = RestCatalog::new(fixture.catalog_config.clone());
+    let ns = apple_ios_ns().await;
```

Comment:
What about creating a per-test randomized namespace instead? I don't get why do we want a shared namespace. I guess before this PR, we just didn't consider the concurrency and conflict problem.

Reply:
Yup, that makes sense, I'll change it.